### ISSUE TITLE
Cannot reproduce COIN result

### ISSUE THREAD

**Issue by nguyentthong** [showlab/videollm-online#41]
hi @chenjoya, I have run your ./scripts/live1+.sh script. This is the training loss landscape:

<img width="1106" alt="image" src="https://github.com/user-attachments/assets/e2c6904a-f0c3-4387-b6f7-9c279c5d70c0">

The loss landscape does not seem to possess any notable spikes. However, I still obtained abnormally low performance:
```
{'eval_coin_step_test_accuracy': 10.315344304495724, 'eval_coin_step_test_runtime': 1433.9594, 'eval_coin_step_test_samples_per_second': 7.585, 'eval_coin_step_test_steps_per_second': 1.897, 'epoch': 4.999838171990808, 'eval_coin_next_test_accuracy': 12.874103388572841, 'eval_coin_next_test_runtime': 1060.116, 'eval_coin_next_test_samples_per_second': 7.627, 'eval_coin_next_test_steps_per_second': 1.907, 'eval_coin_task_test_accuracy': 0.7222824124232575, 'eval_coin_task_test_runtime': 760.7936, 'eval_coin_task_test_samples_per_second': 3.64, 'eval_coin_task_test_steps_per_second': 0.911, 'eval_coin_procedure_test_accuracy': 18.017247054176817, 'eval_coin_procedure_test_runtime': 3311.2593, 'eval_coin_procedure_test_samples_per_second': 2.442, 'eval_coin_procedure_test_steps_per_second': 0.611, 'eval_coin_taskprocedure_test_accuracy': 19.445724442103877, 'eval_coin_taskprocedure_test_runtime': 3289.645, 'eval_coin_taskprocedure_test_samples_per_second': 2.458, 'eval_coin_taskprocedure_test_steps_per_second': 0.615}
```

Can you help me look into the problem? I have examined the evaluation code according to issues #26 and #2.
